jujugui/static/gui/src/app/utils/changes-utils.js

```diff
@@ -314,7 +314,7 @@ YUI.add('changes-utils', function(Y) {
         default:
           changeItem.icon = 'changes-unknown';
           changeItem.description = 'An unknown change has been made ' +
-              'to this enviroment via the CLI.';
+              'to this model via the CLI.';
```

Comment:
If you're not going to add the log generator here can you create a critical issue to do so as it'll need to be done pre-release